com.microsoft.mrtk.input/Utilities/PoseSource/HandJointPoseSource.cs

```diff
@@ -13,6 +13,19 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public class HandJointPoseSource : HandBasedPoseSource
     {
+        /// <summary>
+        /// Empty constructor for inspector instantiation
+        /// </summary>
+        public HandJointPoseSource()
```

Comment:
Format nit, `public HandJointPoseSource() { }` is cleaner

Reply:
Done